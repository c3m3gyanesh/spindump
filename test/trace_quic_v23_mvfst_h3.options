--report-notes --report-spins --format text
