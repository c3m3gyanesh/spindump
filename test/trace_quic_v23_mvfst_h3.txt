A test using the Picoquic client and the Facebook "Mvfst" (Move Fast) QUIC implementation. Using the following commands:

    ./picoquic/picoquicdemo -l ./test.log -L -a "h3-23" -v ff000017 fb.mvfst.net 4433 /5000
    ./picoquic/picoquicdemo -l ./test.log -L -a "h3-23" -v ff000017 fb.mvfst.net 4433 /50000
