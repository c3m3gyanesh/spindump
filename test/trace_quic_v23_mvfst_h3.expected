QUIC 172.16.114.251 <-> 3.15.16.10 088b21ec311a5ed8-fa197e41555fbc16 (50894:4433) at 1571176534750443 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 088b21ec311a5ed8-08400003d1381a5a (50894:4433) at 1571176534832481 measurement up right 82038 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 8b21ec311a5ed86d-08400003d1381a5a (50894:4433) at 1571176534833803 spinvalue up 0 responder packets 1 3 bytes 1280 516 bandwidth 1280 516 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 8b21ec311a5ed86d-08400003d1381a5a (50894:4433) at 1571176534833804 spinvalue up 0 responder packets 1 4 bytes 1280 611 bandwidth 1280 611 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 8b21ec311a5ed86d-08400003d1381a5a (50894:4433) at 1571176534833805 spinvalue up 0 responder packets 1 5 bytes 1280 1891 bandwidth 1280 1891 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 8b21ec311a5ed86d-08400003d1381a5a (50894:4433) at 1571176534834393 spinvalue up 0 responder packets 1 6 bytes 1280 3171 bandwidth 1280 3171 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 8b21ec311a5ed86d-08400003d1381a5a (50894:4433) at 1571176534834397 spinvalue up 0 responder packets 1 7 bytes 1280 4451 bandwidth 1280 4451 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 8b21ec311a5ed86d-08400003d1381a5a (50894:4433) at 1571176534834397 spinvalue up 0 responder packets 1 8 bytes 1280 5731 bandwidth 1280 5731 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 8b21ec311a5ed86d-08400003d1381a5a (50894:4433) at 1571176534834935 spinvalue up 1 initiator packets 2 9 bytes 1504 5933 bandwidth 1504 5933 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 8b21ec311a5ed86d-08400003d1381a5a (50894:4433) at 1571176534914283 spinvalue up 0 responder packets 3 10 bytes 1569 6007 bandwidth 1569 6007 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 08c6ba11c92c72c7-1df22d1c95436571 (61268:4433) at 1571176551886845 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 08c6ba11c92c72c7-0840000347efa4ec (61268:4433) at 1571176551969687 measurement up right 82842 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552056873 spinvalue up 0 responder packets 2 7 bytes 2560 3716 bandwidth 2560 3716 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552066109 spinvalue up 1 initiator packets 3 8 bytes 2784 3805 bandwidth 2784 3805 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552066171 spinvalue up 1 initiator packets 4 8 bytes 2890 3805 bandwidth 2890 3805 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552140406 spinvalue up 0 responder packets 5 9 bytes 4390 3879 bandwidth 4390 3879 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552140768 spinvalue up 1 initiator packets 5 10 bytes 4390 4188 bandwidth 4390 4188 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552150379 spinvalue up 0 responder packets 6 10 bytes 4473 4188 bandwidth 4473 4188 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552150826 spinvalue up 0 responder packets 6 11 bytes 4473 5468 bandwidth 4473 5468 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552150830 spinvalue up 0 responder packets 6 12 bytes 4473 6748 bandwidth 4473 6748 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552150832 spinvalue up 0 responder packets 6 13 bytes 4473 8028 bandwidth 4473 8028 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552150833 spinvalue up 0 responder packets 6 14 bytes 4473 9308 bandwidth 4473 9308 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552150835 spinvalue up 0 responder packets 6 15 bytes 4473 10588 bandwidth 4473 10588 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552150836 spinvalue up 0 responder packets 6 16 bytes 4473 11868 bandwidth 4473 11868 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552150837 spinvalue up 0 responder packets 6 17 bytes 4473 13148 bandwidth 4473 13148 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552150869 spinvalue up 0 responder packets 6 18 bytes 4473 14428 bandwidth 4473 14428 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552150871 spinvalue up 0 responder packets 6 19 bytes 4473 15708 bandwidth 4473 15708 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552150883 spinvalue up 1 initiator packets 6 20 bytes 4473 16988 bandwidth 4473 16988 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552151124 spinvalue up 0 responder packets 7 20 bytes 4556 16988 bandwidth 4556 16988 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552151130 spinvalue up 0 responder packets 7 21 bytes 4556 18268 bandwidth 4556 18268 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552151133 spinvalue up 0 responder packets 7 22 bytes 4556 19548 bandwidth 4556 19548 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552151177 spinvalue up 1 initiator packets 7 23 bytes 4556 20277 bandwidth 4556 20277 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552151361 spinvalue up 1 initiator packets 8 23 bytes 4639 20277 bandwidth 4639 20277 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552151411 spinvalue up 1 initiator packets 9 23 bytes 4722 20277 bandwidth 4722 20277 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552221493 spinvalue up 0 responder packets 10 23 bytes 4805 20277 bandwidth 4805 20277 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552221675 spinvalue up 1 initiator packets 10 24 bytes 4805 20867 bandwidth 4805 20867 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552231592 spinvalue up 0 responder packets 11 24 bytes 4888 20867 bandwidth 4888 20867 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552231948 spinvalue up 0 responder packets 11 25 bytes 4888 22147 bandwidth 4888 22147 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552231955 spinvalue up 0 responder packets 11 26 bytes 4888 23427 bandwidth 4888 23427 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552231957 spinvalue up 0 responder packets 11 27 bytes 4888 24707 bandwidth 4888 24707 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552231958 spinvalue up 0 responder packets 11 28 bytes 4888 25987 bandwidth 4888 25987 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552231960 spinvalue up 0 responder packets 11 29 bytes 4888 27267 bandwidth 4888 27267 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232002 spinvalue up 0 responder packets 11 30 bytes 4888 28547 bandwidth 4888 28547 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232004 spinvalue up 0 responder packets 11 31 bytes 4888 29827 bandwidth 4888 29827 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232006 spinvalue up 0 responder packets 11 32 bytes 4888 31107 bandwidth 4888 31107 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232023 spinvalue up 1 initiator packets 11 33 bytes 4888 32387 bandwidth 4888 32387 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232122 spinvalue up 0 responder packets 12 33 bytes 4971 32387 bandwidth 4971 32387 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232125 spinvalue up 0 responder packets 12 34 bytes 4971 33667 bandwidth 4971 33667 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232127 spinvalue up 0 responder packets 12 35 bytes 4971 34947 bandwidth 4971 34947 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232316 spinvalue up 0 responder packets 12 36 bytes 4971 36227 bandwidth 4971 36227 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232322 spinvalue up 0 responder packets 12 37 bytes 4971 37507 bandwidth 4971 37507 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232324 spinvalue up 0 responder packets 12 38 bytes 4971 38787 bandwidth 4971 38787 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232326 spinvalue up 0 responder packets 12 39 bytes 4971 40067 bandwidth 4971 40067 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232337 spinvalue up 1 initiator packets 12 40 bytes 4971 41347 bandwidth 4971 41347 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232416 spinvalue up 0 responder packets 13 40 bytes 5054 41347 bandwidth 5054 41347 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232420 spinvalue up 0 responder packets 13 41 bytes 5054 42627 bandwidth 5054 42627 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232474 spinvalue up 0 responder packets 13 42 bytes 5054 43907 bandwidth 5054 43907 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232616 spinvalue up 1 initiator packets 13 43 bytes 5054 45187 bandwidth 5054 45187 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232861 spinvalue up 1 initiator packets 14 43 bytes 5137 45187 bandwidth 5137 45187 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552232972 spinvalue up 1 initiator packets 15 43 bytes 5220 45187 bandwidth 5220 45187 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552246627 spinvalue up 0 responder packets 16 43 bytes 5303 45187 bandwidth 5303 45187 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552246633 spinvalue up 0 responder packets 16 44 bytes 5303 46467 bandwidth 5303 46467 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552246635 spinvalue up 0 responder packets 16 45 bytes 5303 47747 bandwidth 5303 47747 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552246708 spinvalue up 0 responder packets 16 46 bytes 5303 49027 bandwidth 5303 49027 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552246712 spinvalue up 0 responder packets 16 47 bytes 5303 50307 bandwidth 5303 50307 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552246714 spinvalue up 0 responder packets 16 48 bytes 5303 51587 bandwidth 5303 51587 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552246821 spinvalue up 0 responder packets 16 49 bytes 5303 52867 bandwidth 5303 52867 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552247100 spinvalue up 1 initiator packets 16 50 bytes 5303 53045 bandwidth 5303 53045 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552247155 spinvalue up 1 initiator packets 17 50 bytes 5386 53045 bandwidth 5386 53045 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552301729 spinvalue up 0 responder packets 18 50 bytes 5469 53045 bandwidth 5469 53045 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552301921 spinvalue up 1 initiator packets 18 51 bytes 5469 54197 bandwidth 5469 54197 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552312663 spinvalue up 0 responder packets 19 51 bytes 5552 54197 bandwidth 5552 54197 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552312668 spinvalue up 0 responder packets 19 52 bytes 5552 55477 bandwidth 5552 55477 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552312909 spinvalue up 0 responder packets 19 53 bytes 5552 56757 bandwidth 5552 56757 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552312959 spinvalue up 1 initiator packets 19 54 bytes 5552 56877 bandwidth 5552 56877 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552313211 spinvalue up 1 initiator packets 20 54 bytes 5635 56877 bandwidth 5635 56877 note "V23,no spin"
QUIC 172.16.114.251 <-> 3.15.16.10 c6ba11c92c72c769-0840000347efa4ec (61268:4433) at 1571176552394455 spinvalue up 0 responder packets 21 54 bytes 5700 56877 bandwidth 5700 56877 note "V23,no spin"
